python/python/res/job_queue/job_manager.py

```diff
@@ -236,11 +235,9 @@ def completeStatus(self, exit_status, error_msg, job=None):
         with open(self.STATUS_file, "a") as f:
             if exit_status == 0:
                 status = ""
-                self.postMessage(job=job, extra_fields=extra_fields)
             else:
                 status = " EXIT: %d/%s" % (exit_status, error_msg)
                 extra_fields.update({"error_msg": error_msg})
-                self.postMessage(job=job, extra_fields=extra_fields)
```

Comment:
We are posting in the "exit" function, which it seems to me is called every time a job fails. Then we also post the stderr